pkg/helpers/helpers.go

```diff
@@ -117,25 +117,18 @@ func CreateSSH(rg io.Reader, s *i18n.Translator) (privateKey *rsa.PrivateKey, pu
 // AcceleratedNetworkingSupported check if the VmSKU support the Accelerated Networking
 func AcceleratedNetworkingSupported(sku string) bool {
 	switch sku {
```

Comment:
I think we want something like this here:

```
// Trim the optional _Promo suffix.
		sku = strings.TrimSuffix(sku, "_Promo")
```

Basically since we're tolerant to the `_Promo` suffix and assume it's the equivalent as the non-promo SKU we want to just strip it if it's present so we can mark equivalent promo SKUs appropriately for accelerated networking support.

Reply:
done